remake_depth.yml

```diff
@@ -40,19 +40,21 @@ targets:
     command: depth_link_mnlakelist("data/depth_data_raw/depth_mn_lakelist.tsv")
     depends:
       - data/NHD_shape_large/NHDWaterbody.dbf
+      #- data/NHD_state_crosswalk/nhd2dowlknum.RData
     
   data/depth_data_linked/depth_herb_jacobson.tsv:
     command: depth_link_jacobson("data/depth_data_raw/depth_Herb_Jacobson.csv")
     depends:
       - data/NHD_shape_large/NHDWaterbody.dbf
+      #- data/NHD_state_crosswalk/nhd2dowlknum.RData
     
   data/depth_data_linked/depth_lagos.tsv:
     command: depth_link_lagos(necsc_config)
     depends:
       - data/NHD_shape_large/NHDWaterbody.dbf
     
   data/depth_data_linked/depth_swimms_wisconsin.csv:
-    command: depth_link_swimms("data/depth_data_raw/wisconsin_swims_lakes_depth.tsv")
+    command: depth_link_swimms("data/depth_data_raw/wisconsin_swims_lakes_depth.tsv", "data/NHD_state_crosswalk/nhd2wbic.RData")
```

Comment:
I think it is handled in the same way as in `depth_link_jacobson`, which is a load of some crosswalk file that is created by some other script that isn't captured in a remake file (e.g., https://github.com/USGS-R/necsc-lake-modeling/blob/daaaa662050bccb006d81dcc45a65c9c1466a4b6/scripts/crosswalk_MN_nhd_dowlknum.R and https://github.com/USGS-R/necsc-lake-modeling/blob/daaaa662050bccb006d81dcc45a65c9c1466a4b6/scripts/depth_link_jacobson.R#L6). I did the pattern here to add that file as a dependency so this would be re-run if the crosswalk file changes. I commented out adding the same dependencies to those other crosswalk links above, but those should probably be uncommented. 

I want to get that hydrolayer crosswalk into a remake (hence my comments about the local file on another issue), but just isn't a priority right now.